### PR TITLE
Addressing accessibility and SEO issues reported by Lighthouse

### LINKED ISSUE #137: Add missing alt texts and/or aria labels for the main page
In order to catch these, recommend running Lighthouse on the page, and seeing which images are missing alt text.

[CodeSee Recording](https://app.codesee.io/shared/0b0b10f0-7490-11ec-a628-61bee2c3b61c)

### PR DESCRIPTION
**Pull request by RabeaGleissner**
This PR addresses this issue: https://github.com/distributeaid/distributeaid.org/issues/137

- added missing alt tags to images (I figured some images were decorative only, so I added an empty alt attribute as per the recommendation in Lighthouse)
- fixed a structural HTML issue where elements other than `<li>`s were inside a `<ul>`
- added the `lang` attribute to the doc

SEO issues:
- added a meta description
- made an anchor text more descriptive

### CODE REVIEW

**Comment by RabeaGleissner** [comment]
Do we need both a `Default` and a `SimpleLayout` component? Do you think it's worth trying to combine those two? That would remove some duplication...

File: content/pages/home.md [hunk 1]
```diff
@@ -5,39 +5,42 @@ missionStatement: |-
...
+    description:
+      Distribute Aid was officially invited to join Operation _Allies Welcome_
+      at Fort Pickett in November. Our role on base is to manage all the donated aid.
+      We work with NGOs and departments across the base to assess the needs of our guests,
+      and work with the community of external organizations supporting the Operation
+      _Allies Welcome_ mission to help facilitate aid deliveries. Once aid arrives on
+      base we work with the US Marines to distribute items to guests, and supply materials
+      for NGOs running education & activity programs to use in their classes. In November
+      & December, our efforts resulted in more than 750,000 items being distributed
+      to or used by 7,500 guests at Fort Pickett. You can support our work by [donating to us through our
```
Comment (RabeaGleissner) [inline content/pages/home.md:44]: The only change in this file is the anchor text for this link here... the rest is probably prettier doing some prettifying... 🤨

Comment (hola-soy-milk) [reply content/pages/home.md:44]: Thank you for running the prettifying!

File: src/components/home/HowWeHelpSection.tsx [hunk 2]
```diff
@@ -14,14 +13,34 @@ export type HowWeHelpBlock = {
...
+    image: covidAidImage,
+    altText: 'People in a room using sewing machines to create masks.',
+  },
+  {
+    image: inKindDonationsImage,
+    altText: 'Two men hugging.',
+  },
+  {
+    image: fortPickettImage,
+    altText:
```
Comment (hola-soy-milk) [inline src/components/home/HowWeHelpSection.tsx:32]: Thank you for adding this!!

**Review by hola-soy-milk — APPROVED** [review]
Thank you so much for fixing these, @RabeaGleissner !

**Pull request merged**